### ISSUE TITLE
Bug: Warning when custom location is specified and unavailable persists when location becomes available

### ISSUE THREAD

**Issue by timboxyz** [jamesmontemagno/MyStreamTimer#68]
On MacOS when using a custom TXT location, if it is unavailable (e.g. folder deleted) a warning box appears.
If the issue is resolved (folder created) the warning still pops up even though MyStreamTimer actually begins its count successfully behind the pop up.
.
This situation persists on an application reboot and can only be fixed by re-selecting the folder.

**Comment by jamesmontemagno** [comment]
Yes, you will need to re-select unfortunately it is a new folder under the hood. :( sandboxes.
